x-pack/plugin/ml/src/main/java/org/elasticsearch/xpack/ml/MachineLearning.java

```diff
@@ -510,7 +511,10 @@ public Map<String, Processor.Factory> getProcessors(Processor.Parameters paramet
 
     // This is not used in v8 and higher, but users are still prevented from setting it directly to avoid confusion
     private static final String PRE_V8_MAX_OPEN_JOBS_NODE_ATTR = "ml.max_open_jobs";
+    public static final String CPU_RATIO_NODE_ATTR = "ml.memory_cpu_ratio";
```

Comment:
Please add a comment here to say that it's gigabytes of memory for this attribute.